cql/DashboardLibrary.cql

```diff
@@ -160,40 +160,49 @@ context Patient
-        when c ~ "Atypical Glandular Cells Uncertain" then {text: 'AGC-US', rank: 1}
-        when c in "AIS" then {text: 'AIS', rank: 1}
-        when c ~ "Endocervical AIS" then {text: 'Endocervical AIS', rank: 1}
-        when c ~ "Endocervical Cells Favor Neoplasia" then {text: 'Endocervical Cells Favor Neoplasia', rank: 1}        
-        when c ~ "Atypical Endometrial Cells" then {text: 'Atypical Endometrial Cells', rank: 1}
-
-        when c ~ "Histiocytes" then {text: 'Histiocytes', rank: 1}
-        when c ~ "Benign Endometrial Cells" then {text: 'Benign Endometrial Cells', rank: 1}
-        when c ~ "Benign Glandular Cells" then {text: 'Benign Glandular Cells', rank: 1}
-        when c ~ "Endometrial Stromal Cells" then {text: 'Endometrial stromal cells', rank: 1}
-        when c ~ "Unsatisfactory" then {text: 'Unsatisfactory', rank: 1}
-        when c ~ "Absent Transformation Zone" then {text: 'Absent Transformation Zone', rank: 1}
-
-        when c is null then {text: 'NULL', rank: -1}
-        else {text: 'UNK', rank: 1}
+        when c ~ "AGC Favor Neoplasia" then {riskTableInput: 'AGC-FN', label: 'AGC-FN', rank: 1, grade: 2}
+        when c ~ "Atypical Endocervical Cells" then {riskTableInput: 'AECC', label: 'Atypical Endocervical Cells', rank: 1, grade: 2}
+        when c ~ "Atypical Glandular Cells Uncertain" then {riskTableInput: 'AGC-US', label: 'AGC-US', rank: 1, grade: 2}
+        when c in "AIS" then {riskTableInput: 'AIS', label: 'AIS', rank: 1, grade: 2}
+        when c ~ "Endocervical AIS" then {riskTableInput: 'ECAIS', label: 'Endocervical AIS', rank: 1, grade: 2}
+        when c ~ "Endocervical Cells Favor Neoplasia" then {riskTableInput: 'ECFN', label: 'Endocervical Cells Favor Neoplasia', rank: 1, grade: 2}
+        when c ~ "Atypical Endometrial Cells" then {riskTableInput: 'AEMC', label: 'Atypical Endometrial Cells', rank: 1, grade: 1}
+
+        when c ~ "Histiocytes" then {riskTableInput: 'HIST', label: 'Histiocytes', rank: 1, grade: 1}
+        when c ~ "Benign Endometrial Cells" then {riskTableInput: 'BEMC', label: 'Benign Endometrial Cells', rank: 1, grade: 1}
+        when c ~ "Benign Glandular Cells" then {riskTableInput: 'BGC', label: 'Benign Glandular Cells', rank: 1, grade: 1}
+        when c ~ "Endometrial Stromal Cells" then {riskTableInput: 'EMSC', label: 'Endometrial stromal cells', rank: 1, grade: 1}
+        when c ~ "Unsatisfactory" then {riskTableInput: 'UNSAT', label: 'Unsatisfactory', rank: 1, grade: 1}
+        when c ~ "Absent Transformation Zone" then {riskTableInput: 'ATZ', label: 'Absent Transformation Zone', rank: 1, grade: 1}
+
+        when c is null then {riskTableInput: 'NULL', label: 'None', rank: -1, grade: 0}
+        else {riskTableInput: 'UNK', label: 'Unrecognized', rank: 1, grade: 0}
       end
   else
-    {{text: 'ALL', rank: 0}}
+    {{riskTableInput: 'ALL', label: 'ALL', rank: 0, grade: 0}}
```

Comment:
Updated label to 'None' for conclusions is null case

Reply:
Sorry I should have said "No result found", since all of the "None" displays were changed to "No result found" in https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/pull/153#discussion_r1881225503 (I think I made this comment before that one). Could we add that quick change to these displays too, for consistency?

Reply:
Updated to No result found.